goodtablesio/plugins/github/tasks.py

```diff
@@ -43,8 +43,14 @@ def get_validation_conf(clone_url, job_id):
 
 @signals.task_postrun.connect(sender=validate)
 def post_task_handler(**kwargs):
+    # We need to import the DB connection at this point, as it has been
+    # initialized when the worker started
+    from goodtablesio.tasks import tasks_db
 
-    job = kwargs['retval']
+    # We can't use retval because on errors
+    # it will be an exception
+    job_id = kwargs['kwargs']['job_id']
```

Comment:
Good idea. I'll update. Though I don't think db calls will be a bottleneck at any point of the project.